frontend/public/components/dashboard/details-card/detail-item.tsx

```diff
@@ -3,17 +3,31 @@ import classNames from 'classnames';
 
 import { LoadingInline } from '../../utils';
 
-export const DetailItem: React.FC<DetailItemProps> = React.memo(({ title, value, isLoading }) => (
-  <React.Fragment>
-    <dt className="co-details-card__item-title">{title}</dt>
-    <dd className={classNames('co-details-card__item-value', {'text-secondary': !value})}>
-      {isLoading ? <LoadingInline /> : value || 'Unavailable'}
-    </dd>
-  </React.Fragment>
-));
+export const DetailItem: React.FC<DetailItemProps> = React.memo(
+  ({ title, value, isLoading, children, error = false }) => {
```

Comment:
hm, seems to me that the contract is a bit unclear. I would remove the `value` prop and just keep the `children`. I know it will mean that we need to update all usages, but we dont have that many yet so better do it sooner than later.

Reply:
+1 removing `value` . Using children sounds good for future changes like this